modules/programs/looking-glass-client.nix

```
@@ -0,0 +1,60 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.looking-glass-client;
+  settingsFormat = pkgs.formats.ini { };
+in {
+  meta.maintainers = with maintainers; [ j-brn ];
+
+  options.programs.looking-glass-client = {
+    enable = mkEnableOption "looking-glass-client";
+
+    package = mkPackageOption pkgs "looking-glass-client" { };
+
+    settings = mkOption {
+      type = settingsFormat.type;
+      default = { };
+      description = "looking-glass-client settings";
```

```suggestion
      description = "looking-glass-client settings.";
```
Missed the final dot.